src/reducers/scenes/WalletsReducer.ts

```diff
@@ -6,44 +6,9 @@ import { Action } from '../../types/reduxTypes'
 export interface WalletsState {
   selectedWalletId: string
   selectedCurrencyCode: string
-  walletLoadingProgress: { [walletId: string]: number }
   fioWallets: EdgeCurrencyWallet[]
 }
 
-const walletLoadingProgress = (state = {}, action: Action): WalletsState['walletLoadingProgress'] => {
-  switch (action.type) {
-    case 'INSERT_WALLET_IDS_FOR_PROGRESS': {
```

Comment:
You need to visit reduxActions.ts and delete the entries for `INSERT_WALLET_IDS_FOR_PROGRESS`, `UPDATE_WALLET_LOADING_PROGRESS`, and `RESET_WALLET_LOADING_PROGRESS`. You may find a few more places that need to be cleaned up, once the types are gone.

Reply:
I didn't do this for `INSERT_WALLET_IDS_FOR_PROGRESS` because there are still users of `activeWalletIds`

Seems safe to remove the others, though.